frontend/dockerfile/dockerfile2llb/convert.go

```diff
@@ -162,6 +162,67 @@ func ListTargets(ctx context.Context, dt []byte) (*targets.List, error) {
 	return l, nil
 }
 
+func parseLintOptions(checkStr string) (*linter.Config, error) {
```

Comment:
In future I think this can be moved to public function in linter pkg (and called both from here in dockerui) but that can be changed in some follow-up PR.